doc/author_williams.txt

```diff
@@ -0,0 +1 @@
+I hereby place my contributions to libsc into the public domain under FreeBSD license.  Kyle Williams.
```

Comment:
Thanks; we've accepted this formulation in the past, but public domain and FreeBSD are not exactly the same. If you were to choose only the latter, that would be perfect.

Reply:
Of course, I just copied one of the other ones.